backend/db/migrate/20240602075324_create_boards.rb

```diff
@@ -0,0 +1,13 @@
+class CreateBoards < ActiveRecord::Migration[7.0]
+  def change
+    create_table :boards do |t|
+      t.integer :song_id, :null => false #曲id
```

Comment:
```suggestion
      t.belongs_to :song  #曲id
```
上記の様に直すと、songに対して、リレーションが引けます！
https://railsguides.jp/association_basics.html#has-many%E9%96%A2%E9%80%A3%E4%BB%98%E3%81%91

Reply:
修正しました
https://github.com/yura-8/Let-s-Live-go-server/commit/996d3378ef1477aabe8c2dbd215108728d15deea